features/works/work_download.feature

```diff
@@ -5,180 +5,172 @@
 Feature: Download a work
   @wip
```

Comment:
Github has no suitable emoji. Oh well, something to add to the list of things to fix some day.